components/brave_account/core/mojom/brave_account.mojom

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+module brave_account.mojom;
+
+interface BraveAccountHandlerFactory {
+  CreateBraveAccountHandler(pending_receiver<BraveAccountHandler> handler,
+                            pending_remote<BraveAccountClient> client);
+};
```

Comment:
Do we even need a Factory - won't there only be a single `BraveAccountHandler` for a WebUI?

Reply:
agreed :)

Reply:
The factory is there to ensure that `BraveAccountHandler` and `BraveAccountClient` are always created together. It is true, though, that if we remove `BraveAccountClient`, then there's no point in having the factory.